client/state/plugins/marketplace/actions.ts

```diff
@@ -40,3 +67,168 @@ export function setIsPluginInstalledDuringPurchase(
 		isPluginInstalledDuringPurchase,
 	};
 }
+
+export function resetPurchaseFlow(): { type: string } {
+	return {
+		type: MARKETPLACE_RESET_PURCHASE_FLOW,
+	};
+}
+
+export function siteTransferStateChange(
+	state: MARKETPLACE_ASYNC_PROCESS_STATUS,
+	reason = ''
+): { type: string; state: MARKETPLACE_ASYNC_PROCESS_STATUS; reason: string } {
+	return {
+		type: MARKETPLACE_SITE_TRANSFER_STATE_CHANGE,
+		state,
+		reason: reason ?? 'NOT_PROVIDED',
```

Comment:
Seems like we're emulating a default value here. Should we move the `NOT_PROVIDED` to the default value in the definition of this action creator?